### DISCUSSION TITLE
Infinite loading

### DISCUSSION THREAD

**Discussion by MujibBasha** [codemagic-ci-cd/codemagic-docs#1672] [Q&A] [answered]
![image](https://github.com/codemagic-ci-cd/codemagic-docs/assets/74606434/717053ad-6903-4a33-a28d-e0299d547d90)
![image](https://github.com/codemagic-ci-cd/codemagic-docs/assets/74606434/5fe77120-756c-4f3f-a7c8-c9c7a019fccf)

**Comment by dtrdic** [comment, marked as answer]
Hey @MujibBasha can you please try it right now? should be ok

**Comment by MujibBasha** [comment]
Thanks , Know is working
